### ISSUE TITLE
Identical playlist items in sequence in a schedule do not reload

### ISSUE THREAD

**Issue by PatrickA-RiseVision** [Rise-Vision/rise-launcher-electron#841]
**Describe the bug**
If I place identical playlist items in sequence in a schedule, the item is not reloaded when the duration is elapsed.  This is most obvious when using something like Google Slides Template, and then making a change to the Google Slides.  I can also replicate this using URLs, if it's the same one twice.

**Reach**
Any user who is using Google Slides template and updates their content frequently will notice this.

**Impact**
This is preventing the workaround for a limitation in the Google Slides template from working as expected.  The only way to work around it is to schedule a different Template to play immediately after the Google Slides Template, which isn't very practical.

**To Reproduce**
1. Create a presentation using the Google Slides template.
2. Provide the URL for a Google Slides document you can edit.
3. Create a schedule, and add the Google Slides template to the schedule twice.
4. Play the schedule on a Display.
5. Make a change to your Google Slides document.

**Expected behavior**
Templates with third party content are reloaded each time they are played.  In the case of Google Slides, this was previously the only way to have the Template update with new content from Slides.

**Screenshots**
here's a quick video demonstrating it: https://drive.google.com/a/risevision.com/file/d/1ZPO79jJWnVmjF8ijUWWItBBgtv9m0Qr9/view?usp=drivesdk

**Desktop (please complete the following information):**
 - OS: Windows, Linux.

**Additional context**
This is preventing users from using the workaround we've been proposing for the Google Slides Templates' shortcomings.

**Comment by andrecardoso** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Defect #841 Identical playlist items in sequence in a schedule do not reload (2)](https://trello.com/c/PT63n4ey/435-defect-841-identical-playlist-items-in-sequence-in-a-schedule-do-not-reload-2)

**Comment by olegrise** [comment]
@PatrickA-RiseVision @sheadarlison should we close this bug considering that this problem is going to be addressed by "Google Slides Play Until Done" epic in Q7?

**Comment by sheadarlison** [comment]
@PatrickA-RiseVision does this only impact Google Slides?

**Comment by PatrickA-RiseVision** [comment]
@sheadarlison @andrecardoso this happens regardless whether it's google slides.  If the same template appears twice in a row in a schedule, player doesn't reload it.

it can't wait until Q7.  this is preventing us from working around the Google Slides problem among other things.

**Comment by sheadarlison** [comment]
@PatrickA-RiseVision okay, thanks

@andrecardoso @olegrise given this impacts all templates I think it needs to be fixed.

**Comment by olegrise** [comment]
@PatrickA-RiseVision I think we are mixing two different problems into one issue.

**Problem 1** - Google Slides do not refresh after making a change to the Google Slides document. This is going to be addressed as part of the upcoming epic.

**Problem 2** - When you add the URL item to the schedule, then webpage does not reload. This problem has nothing to do with the Templates.

If above is correct then I have 2 questions: 
- How big is the second problem? The reason I ask is because reloading URL item on every iteration may create another problem which is inefficient use of resources and bad user experience - imaging a heavy or slow website being reloaded every 10 seconds.
- How this problem should be fixed -reload on every iteration; allow user to configure reload rules; something else?

**Comment by PatrickA-RiseVision** [comment]
@olegrise 

"Problem 1" is not what I'm talking about, with the exception that we have been sharing a workaround with users for some time which no longer works suddenly.

"Problem 2", I was able to replicate this using templates.  I used Google Slides template as the example because it's the most obvious use case for having the same Presentation twice in a row cause a reload, and the easiest to demonstrate with.

Until recently, we were sharing this as a workaround for the issue wherein Google Slides does not refresh when updated.  Suddenly the behavior appears to have changed.  I now have 2 tickets blocked on it, since it stopped working as expected and the workaround broke.

Worth noting too that this behaves as you'd expect if you put 2 different presentations one after another in a schedule playlist.  The only time we forego reloading for efficiency appears to be if the same presentation is scheduled twice in a row.
